test/driver.js

```diff
@@ -428,569 +376,550 @@ var Driver = (function DriverClosure() {
     this.canvas = document.createElement("canvas");
   }
 
-  Driver.prototype = {
-    _getQueryStringParameters: function Driver_getQueryStringParameters() {
-      const queryString = window.location.search.substring(1);
-      return Object.fromEntries(new URLSearchParams(queryString).entries());
-    },
+  _getQueryStringParameters() {
```

Comment:
Should we perhaps export https://github.com/mozilla/pdf.js/blob/dc455c836e80799fccd8f1741eb8f62e9ec01c37/web/ui_utils.js#L183-L194 in the viewer components and re-use that in this file rather than essentially re-implementing that here?

Reply:
For now I skipped that because the series is already quite big and I intended to only touch the driver code here. We can do that in a follow-up though because I don't see other limitations for us to do that.